modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,204 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-top-aware_{context}"]
+= vSphere Topology Awareness
+
+{product-title} provides the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple data centers and clusters, thus helping to avoid a single-point-of-failure. 
+
+This is accomplished by defining zone and region categories in vCenter, and then assigning these categories to different failure domains, such as a compute cluster, host, or data center, by creating tags for these zone and region categories. After you have created the appropriate categories, and assigned tags to vCenter objects, you can create additional machinesets that create virtual machines (VMs) that are responsible for scheduling pods in those failure domains.
+
+.Procedure
+. In the VMware vCenter vSphere client GUI, define appropriate zone and region catagories and tags.
++
+Zones and regions in the vSphere environment are typically defined by creating categories called `openshift-zone` and `openshift-region`.
+Usually, category names can be any valid string, but {product-title} recommends use of `openshift-zone` and `openshift-region`
+categories. 
++
+The following example defines two failure domains with one region and two zones:
++
+.vSphere topology with one region and two zones
+|===
+|Compute cluster | Failure domain |Description
+
+|Compute cluster: ocp1, 
+Data center: Atlanta
+|openshift-region: us-east-1 (tag), openshift-zone: us-east-1a (tag)
+|This defines a failure domain in region us-east-1 with zone us-east-1a.
+
+|Computer cluster: ocp2, 
+Data center: Atlanta
+|openshift-region: us-east-1 (tag), openshift-zone: us-east-1b (tag)
+|This defines a different failure domain within the same region called us-east-1b.
+|===
++
+For more information about vSphere categories and tags, see the VMware vSphere documentation.
+
+. To allow the container storage interface (CSI) driver to detect this topology, edit the `clusterCSIDriver` object YAML file `driverConfig` section:
+* Specify the `openshift-zone` and `openshift-region` categories that you created earlier.
+* Set `driverType` to `vSphere`.
++
+[source, terminal]
+----
+~ $ oc edit clustercsidriver csi.vsphere.vmware.com -o yaml
+----
++
+Example output:
```

Comment:
This was brought to my attention a few weeks ago, and I don't think we're super consistent with it, but apparently "Example output" needs to be written as follows: 

.Example output 

See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output